cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java

```diff
@@ -131,30 +130,27 @@ public Event intercept(Event event) {
             return events;
         } else {
             List<Event> interceptedEvents = new ArrayList<>(events.size());
-        
+
             for (Event event : events) {
                 Event interceptedEvent = intercept(event);
                 interceptedEvents.add(interceptedEvent);
             } // for
- 
+
             return interceptedEvents;
         } // if else
     } // intercept
 
     @Override
     public void close() {
         /*
```

Comment:
I understand this is a leftover a commneted code should be removed.

Reply:
NTC: There is not a leftover, is just an empty line.

Reply:
fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1529/files#diff-0b9888490f0a264991153957f9271deb